resources/assets/less/AdminLTE.less

```diff
@@ -1,61 +1,72 @@
 /*!
- *   AdminLTE v2.3.0
+ *   AdminLTE v2.4.8
```

Comment:
Updated to new version of Admin LTE and importing the file from the node directory.